contrib/segyconverter/utils/normalize_cube.py

```diff
@@ -0,0 +1,150 @@
+# pylint: disable=import-error
+
+"""
+Utility Script to normalize one cube
+"""
+
+import numpy as np
+import argparse
+
+
+def compute_statistics(stddev: float, mean: float, max_range: float, k: int):
+    """
+        Compute min_clip, max_clip and scale values based on provided stddev, max_range and k values
+        :param stddev: standard deviation value
+        :param max_range: maximum value range
+        :param k: number of standard deviation to be used in normalization
+        :returns: min_clip, max_clip, scale: computed values
+        :rtype: float, float, float
+    """
+    min_clip = mean - k * stddev
+    max_clip = mean + k * stddev
+    scale = max_range / (max_clip - min_clip)
+    return min_clip, max_clip, scale
+
+
+def norm_value(v: float, min_clip: float, max_clip: float, min_range: float, max_range: float, scale: float):
+    """
+        Normalize seismic voxel value to be within [min_range, max_clip] according to
+        statisctics computed previously
+        :param v: value to be normalized
+        :param min_clip: minimum value used for clipping
+        :param max_clip: maximum value used for clipping
+        :param min_range: minium range value
+        :param max_range: maximum range value
+        :param scale: scale value to be used for normalization
+        :returns: normalized value, must be within [min_range, max_range]
+        :rtype: float
+    """
+    offset = 0
+    if scale != 1:
+        offset = -1 * min_clip  # Normalizing - set values between 0 and 1
+    # Clip value
+    if v > max_clip:
+        v = max_clip
+    if v < min_clip:
+        v = min_clip
+    # Scale value
+    v = (v + offset) * scale
+    # This value should ALWAYS be between min_range and max_range here
+    if scale != 1 and (v > max_range or v < min_range):
+        raise Exception('normalized value should be within [{0},{1}].\
+             The value was: {2}'.format(min_range, max_range, v))
+    return v
+
+
+def clip_file(local_filename, stddev, mean, k, min_range_value, max_range_value, normalize=True):
+    """
+        Normalize npy array file according to statistics. This function overwrites the existing
+        data file
+        :param str local_filename: npy data file
+        :param float stddev: standard deviation value
+        :param int k: number of standard deviation to be used in normalization
+        :param float min_range_value: minium range value
+        :param float max_range_value: maximum range value
+    """
+    # Load local npy file
+    cube = np.load(local_filename)
+    # Normalize block
+    try:
+        norm_cube = main(cube=cube, stddev=stddev, mean=mean, k=k, min_range=min_range_value,
```

Comment:
I would suggest change code to avoid call main function here. I can work on this if you agree @MJZawacki